node/nodeTransactions.go

```diff
@@ -79,6 +79,17 @@ func (n *Node) lookupHistoricalTransaction(hash []byte, withResults bool) (*tran
 
 	putMiniblockFieldsInTransaction(tx, miniblockMetadata)
 
+	if ok := (&transaction.StatusComputer{
```

Comment:
this is ugly. Please split in multiple instructions. Why aren't we using a constructor for the StatusComputer? If someone else happens to forget to pass a field, the code will simply panic.

Reply:
added ticket [EN-8653] to refactor this component